lib/pbench/server/api/__init__.py

```diff
@@ -144,6 +148,17 @@ def create_app(server_config):
 
     app.logger = get_pbench_logger(__name__, server_config)
 
+    # sets up email configuration
+    mail_settings = {
```

Comment:
I would like to see the send-mail configuration completely confined either to your new `ForgotPassword` API class or to a new separate class rather than accumulating all sort of unrelated stuff to the Flask app configuration class here.